.github/workflows/build.yml

```diff
@@ -33,7 +33,9 @@ jobs:
         ./gradlew publishToMavenLocal
         cd ..
     - name: Build with Maven
-      run: mvn install
+      run: |
+        mvn install
+        mvn editorconfig:check
```

Comment:
@dr0i This seems not only check the files in the repo but also the files downloaded or created by this action. This leads to failing GH-Action Build. Could you have a look.